webextension/js/background/assignManager.js

```diff
@@ -270,19 +293,37 @@ const assignManager = {
     if (siteSettings === false) {
       return false;
     }
-    // ✓ This is to mitigate https://bugzilla.mozilla.org/show_bug.cgi?id=1351418
-    let prefix = "   "; // Alignment of non breaking space, unknown why this requires so many spaces to align with the tick
+    let checked = true;
```

Comment:
This default needs to be `false` or the menu shows the site assigned even when it's not.

Reply:
True dat thanks